src/controls/src/flatmesh.h

```diff
@@ -57,10 +57,15 @@ class FlatMesh : public QQuickItem
 signals:
     void animatedChanged();
 
+private slots:
+    void onVisibleChanged() { maybeEnableAnimation(); }
```

Comment:
Instead of introducing this slot, you could make maybeEnableAnimation() a slot and connect to it directly.